Makefile

```diff
@@ -134,7 +134,7 @@ test-oci-tar-builder:
 .PHONY: install install-%
 install: $(RUNTIMES:%=install-%);
 
-install-%:
+install-%: build-%
```

Comment:
I think the dependency on `build` was removed from https://github.com/containerd/runwasi/pull/581

Reply:
removed it